src/csharp/Microsoft.Spark/Sql/Types/DataType.cs

```diff
@@ -69,7 +69,7 @@ public string Json
         internal virtual object JsonValue => TypeName;
         
         /// <summary>
-        /// Parses a JSON string to create a JvmObjectReference object.
+        /// Parses a JSON string to create a JvmObjectReference object, which references a StructType object on the JVM side.
```

Comment:
this will be over the number of charcacters limit

Reply:
Can you add the following xml tags as well in your description ?
```xml
<see cref="JvmObjectReference"/>
<see cref="StructType"/>
```

Reply:
Updated. Thanks.